lib/python/qmk/info.py

```diff
@@ -128,17 +128,13 @@ def _extract_features(info_data, rules):
             key = '_'.join(key.split('_')[:-1]).lower()
             value = True if value.lower() in true_values else False if value.lower() in false_values else value
 
-            if 'config_h_features' not in info_data:
```

Comment:
This is actually publicly available at <https://keyboards.qmk.fm/v1/keyboards/0_sixty/info.json>. Assumptions on its use only within the codebase is not enough to ensure it should be removed.

Reply:
On the other side: Public availability proves nothing when it comes without any documentation or usage hints in repository content or history. Any use of this seems to be based on random assumptions, and it seems to be completely fair game to change without notice.

Reply:
Thats not the contract we follow. Its inclusion is almost certainly not going to,
* be merged directly to master
* merged as part of a collection of fixes

Reply:
Ok. I dropped that changeset.

But AFAICS, the code I was trying to remove seems like tech debt. Someone who knows the intent should clarify the purpose. Without clarification, the semantics are unknown, and nobody can ever touch that code without breaking it. Especially the two first lines seems 100% pointless and thus impossible to maintain.